### PR TITLE
Lauren's answer to the Prime Challenge 📓

### PR DESCRIPTION
**Pull request by harkshark**
I know it's a bit late, but wanted to submit my answer to the Prime Challenge via JavaScript!

If you'd like to pull this down to your computer and run it yourself, let me know!

kermit-typing

### CODE REVIEW

File: assignments/submissions/prime-challenge-lauren.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    if (remainder === 0) {
+      return false;                   // not prime, it can be divided evenly into another number
+    }
+  }
+
+  return number !== 1;                // if we've gotten here, it's prime (unless the number is 1)
+}
+
+
+// Instead of asking for a number, check all numbers!
```
Comment (gnklausner) [inline assignments/submissions/prime-challenge-lauren.js:18]: _Aaaaaaaaall_ the numbers? :trollface:

File: assignments/submissions/prime-challenge-lauren.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+// Prime Challenge
+// Given a number between 1-100, return true if the number is a prime.
+
+function isNumberPrime(number) {
+
+  for (var i = 2; i < number; i++) {  // start at 2 because 1 doesn't impact the definition of prime
```
Comment (gnklausner) [inline assignments/submissions/prime-challenge-lauren.js:6]: Can you think of a potential, simple optimization here? 🤔

Comment (LindseyMaddox) [reply assignments/submissions/prime-challenge-lauren.js:6]: #mathnerd

**Comment by LindseyMaddox** [comment]
I love that you used a function within a function to accomplish getting all the primes. This is a great application of what we went over today!

**Pull request closed without merging**